modules/articles/client/controllers/list-articles.client.controller.js

```diff
@@ -10,6 +10,39 @@
   function ArticlesListController(ArticlesService) {
     var vm = this;
 
-    vm.articles = ArticlesService.query();
+    vm.error = null;
+    vm.find = find;
+
+    init();
```

Comment:
You're right. We should use `activate()`. I looked into using Angular 1.5's `$onInit`. But I didn't see how it would work in this scenario. Also, I didn't want to over complicate this PR.